src/plugins/embeddable/public/embeddable_panel/embeddable_panel.tsx

```diff
@@ -207,7 +221,13 @@ export const EmbeddablePanel = (panelProps: UnwrappedEmbeddablePanelProps) => {
           </EuiFlexItem>
         </EuiFlexGroup>
       )}
-      <div className="embPanel__content" ref={embeddableRoot} {...contentAttrs}>
+      {!initialLoadComplete && <PanelLoader />}
+      <div
+        css={{ display: initialLoadComplete ? 'block' : 'none !important' }}
```

Comment:
nit: I think this can be collapsed into:
```suggestion
        css={initialLoadComplete ? undefined : { display: 'none !important' }}
```

Reply:
Good suggestion!